cherrypy/_cpwsgi.py

```diff
@@ -75,11 +75,17 @@ class VirtualHost(object):
     """
 
     def __init__(self, default, domains=None, use_x_forwarded_host=True):
+        """ Override the Object __init__ function to define custom attributes
+        and parameters.
+        """
         self.default = default
         self.domains = domains or {}
         self.use_x_forwarded_host = use_x_forwarded_host
 
     def __call__(self, environ, start_response):
+        """ Override the default __call__ function to define custom attributes
```

Comment:
This is the WSGI callable. Good docstring should give a good prompt to stranger explaining the function's high-level purpose. It should say what behavior/result it would give when called. Optionally, explain the expected argument.
It must also be PEP257-complaint.